.github/workflows/provisioning-tests.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         # 2023-09-28: google-cloud-sdk removed from this list because it was intermittently
         #   unavailable as an apt package to remove, and might be migrating to snap. If more
         #   disk space is needed, see if the snap is installed, and remove that.
-        run:  sudo apt remove --purge -y ghc-* azure-cli hhvm llvm-* dotnet-* powershell mono-* php* ruby*
```

Comment:
hhvm is no longer installed by default in ubuntu 24.04, which is why this was removed.